js/filter.js

```diff
@@ -0,0 +1,97 @@
+import { addAdsToMap } from './map.js';
+
+const LOW_PRICE = 10000;
+const HIGH_PRICE = 50000;
+const RENDER_DELAY = 500;
+const adFilterElement = document.querySelector('.map__filters');
+
+function getStringPriceValue(number) {
+  if(number <= LOW_PRICE) {
+    return 'low';
+  } else if(number >= HIGH_PRICE) {
+    return 'high';
+  }
+
+  return 'middle';
+}
+
+function getCurrentFiltersValues() {
+  const type = document.querySelector('#housing-type').value;
+  const price = document.querySelector('#housing-price').value;
+  const numberRooms = document.querySelector('#housing-rooms').value;
+  const numberGuests = document.querySelector('#housing-guests').value;
+  const features = [];
+  const housingFeatures = document.querySelectorAll('.map__checkbox');
+
+  housingFeatures.forEach((feature) => {
+    if(feature.checked) {
+      features.push(feature.value);
+    }
+  });
+
+  if(features.length === 0) {
+    features.push('any');
+  } else {
+    features.sort((firstFeature, secondFeature) => (firstFeature > secondFeature) ? 1 : -1);
+  }
+
+  return [type, price, numberRooms, numberGuests, features.join('')];
```

Comment:
```suggestion
  function getFeatures() {
    const selectedFeatures = document.querySelectorAll('.map__checkbox:checked');
    
    if (selectedFeatures.length > 0) {
      return Array.from(selectedFeatures).map(f => f.value).sort().join('');
    }
    
    return ['any'];
  }

  return [type, price, numberRooms, numberGuests, getFeatures()];
```

1. You can use `*:checked` selector to get only checked inputs
2. You can use `map` method to make code more simple and readable
3. You don't need pass sort function to `sort` method if you need sort strings alphabetically
4. **Are you sure you need use `join('')` at the end?** At the end it will be long string with no delimiters, is it correct?